mysql-test/suite/sys_vars/t/log_slow_sp_statements_basic.test

```diff
@@ -1 +1,50 @@
+--echo default
+SELECT @@global.log_slow_sp_statements;
+
+--echo --log-slow-sp-statements
+--exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--shutdown_server 10
+--source include/wait_until_disconnected.inc
+--enable_reconnect
+--exec echo "restart:--log-slow-sp-statements" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--source include/wait_until_connected_again.inc
+SELECT @@global.log_slow_sp_statements;
+
+--echo --skip-log-slow-sp-statements
+--exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--shutdown_server 10
+--source include/wait_until_disconnected.inc
+--enable_reconnect
+--exec echo "restart:--skip-log-slow-sp-statements" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
```

Comment:
It is quite unusual for a sys_vars/*_basic test to restart the server with different options. I'd have created a separate test case that has -master.opt with --skip-foo and single SELECT in the testcase. Up to you if you want to make this change.